test/parallel/test-file-write-stream3.js

```diff
@@ -116,13 +113,12 @@ function run_test_2() {
 
 
 function run_test_3() {
-  var file, options;
 
   const data = '\u2026\u2026';    // 3 bytes * 2 = 6 bytes in UTF-8
 
-  options = { start: 10,
+  const options = { start: 10,
               flags: 'r+' };
```

Comment:
Can you line this up so that start and flags line up like they did before (ie need to indent flags to align with new position of start)